### ISSUE TITLE
Exam 19 Question 29

### ISSUE THREAD

**Issue by FreneticXO** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#141]
Correct Answer: B

**Comment by emoleumassi** [comment]
correct. Performance efficiency is the ability to use resources effectively and reliability the ability to recover from disruptions. By using multiple AZs with protect the instance for failure.
